cdk/checkfiles_runner/lambdas/create_instance/main.py

```diff
@@ -0,0 +1,70 @@
+import os
+import base64
+import boto3
+
+
+def get_ami_id():
+    return os.environ['AMI_ID']
+
+
+def get_instance_name():
+    return os.environ['INSTANCE_NAME']
+
+
+def get_instance_type():
+    return os.environ['INSTANCE_TYPE']
+
+
+def create_checkfiles_instance(event, context):
+    instance_name = get_instance_name()
+    ami_id = get_ami_id()
+    instance_type = get_instance_type()
+    # clone checkfiles code and build the virtual environment
+    user_data = '''#!/bin/bash
+    cd /home/ubuntu
+    git clone https://github.com/IGVF-DACC/checkfiles.git
```

Comment:
Expect this might want to be branch specific at some point for spinning up new test versions of CF?

Reply:
Yea, that is a good point. Will add branch to config.